srcpkgs/apr/template

```diff
@@ -29,14 +29,21 @@ if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 	LDFLAGS="-latomic"
 fi
 
+do_check() {
+	cp "${FILESDIR}/etc-services" "${XBPS_MASTERDIR}/etc/services"
+	make test
+}
+
+post_extract() {
+	if [ "${XBPS_TARGET_LIBC}" = "musl" ]; then
+		patch -d "${wrksrc}" -Np1 <"${FILESDIR}/musl-dso.patch"
+	fi
+}
```

Comment:
does this patch also work on glibc? If yes, we should put it to patches/ and not apply it manually here

Reply:
this patch isn't really correct, the reason the test fails is because dlclose is a no-op and doesn't actually unload anything: https://wiki.musl-libc.org/functional-differences-from-glibc.html#Unloading-libraries

the correct thing to do is skip the test on musl

Reply:
@nekopsykose That's very useful info about musl's `dlclose`. I couldn't wrap my head around why it didn't appear to unload the lib. I've now patched out the tests instead. Thank you.

Reply:
@paper42 The patch is only _applicable_ to musl, but I guess it would work on glibc too. Would it be preferable to apply the patch unconditionally?